packages/mobile/src/home/WalletHome.tsx

```diff
@@ -116,6 +128,16 @@ export class WalletHome extends React.Component<Props, State> {
     // Waiting 1/2 sec before triggering to allow
     // rest of feed to load unencumbered
     setTimeout(this.tryImportContacts, 500)
+
+    this.setState({
+      isAccountBalanceZero: !ALL_CURRENCIES.some((currency) =>
+        this.props.balances[currency]?.isGreaterThan(0)
```

Comment:
I think it's worth discussing with Aleksey if `0` is the right number here. My thought is that if someone has a super small balance (e.g., less than 0.01), then we probably still want to show them this banner